modules/goldbachBidAdapter.js

```diff
@@ -0,0 +1,1153 @@
+import { Renderer } from '../src/Renderer.js';
+import * as utils from '../src/utils.js';
```

Comment:
Can you just import the needed functions from 'utils'? A month or so ago we went through all adapters and removed the previous convention of "import *"

Reply:
done :)